packages/cohort/src/state/postgres/database.rs

```diff
@@ -30,7 +30,7 @@ impl Database {
             recycling_method: deadpool_postgres::RecyclingMethod::Fast,
         });
         let pc = PoolConfig {
-            max_size: 100,
+            max_size: 40,
```

Comment:
? :)

Reply:
The max `available` connections on my local pg was around 80ish, and therefore I just used the number in my local. 

One way to fix this is increase the `mac_connection` in my local. But since this code is becoming obsolete soon, I will ignore this for now